protocol/daemons/flags/flags.go

```diff
@@ -11,8 +15,9 @@ const (
 	// Flag names
 	FlagUnixSocketAddress = "unix-socket-address"
 
-	FlagPriceDaemonEnabled     = "price-daemon-enabled"
-	FlagPriceDaemonLoopDelayMs = "price-daemon-loop-delay-ms"
+	FlagPriceDaemonEnabled                = "price-daemon-enabled"
+	FlagPriceDaemonLoopDelayMs            = "price-daemon-loop-delay-ms"
```

Comment:
Meta: I understand that this is a flag that affects the entire price daemon (not just the exchange). 

1. Do you know if this config is used? Is there a need for this flag?
2. Can we move this config under the override flag? I think it's better to minimize the number of flags that we support. This should be standardized across all daemons (liquidations + bridge)

This can be a separate ticket / PR to keep this work focused on the override for price exchanges